### PR TITLE
NEPT-1269: Move helper functions outside of install file.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Review ec_embedded_video:
Could you please apply our guideline to move helper functions outside of the install file to a module_name.install.inc?

### Change log

- Added:
- Changed: Move helper functions outside of install file.
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
No commands

```

### CODE REVIEW

File: profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install [hunk 1]
```diff
@@ -4,6 +4,8 @@
  * Install, update and uninstall functions for the ec_embedded_video module.
  */
 
+module_load_include('inc', 'nexteuropa_multilingual', 'includes/nexteuropa_multilingual.helpers');
```
Comment (Fefaine) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install]: You are not including the right file i think
ec_embedded_video/includes/ec_embedded_video.helpers.inc

Comment (voidtek) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install]: sorry, you are alright.

**Comment by imanoleguskiza** [comment]
:

**Comment by netlooker** [comment]
:+1:

**Comment by imanoleguskiza** [comment]
👍

**Pull request merged**